presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -85,7 +87,9 @@ public CachingJdbcClient(JdbcClient delegate, Set<SessionPropertiesProvider> ses
         this.cacheMissing = cacheMissing;
 
         CacheBuilder<Object, Object> cacheBuilder = CacheBuilder.newBuilder()
-                .expireAfterWrite(metadataCachingTtl.toMillis(), TimeUnit.MILLISECONDS);
+                .expireAfterWrite(metadataCachingTtl.toMillis(), TimeUnit.MILLISECONDS)
```

Comment:
nit: static import

Reply:
I didn't change that line so I'll leave it here.